.changeset/tasty-boxes-brake.md

```diff
@@ -0,0 +1,5 @@
+---
+"@firebase/firestore": patch
+---
+
+Fix for Issue #8474 - Prevent a possible condition of slow snapshots, caused by a rapid series of document update(s) followed by a delete.
```

Comment:
nit: You don't need to specify "Fix for XXXX" as that should be automatically included in the release notes by parsing the PR description. Just make sure the PR description includes the line `Fixes #8474`

Reply:
Thanks for the tip. I've updated and resolving this conversation.